src/api/pastel-rpc/transactions.ts

```diff
@@ -0,0 +1,146 @@
+import hex from 'hex-string'
```

Comment:
Since this came from the `legacy` folder, can we refactor it, so it's not legacy anymore and add tests (example here: https://github.com/pastelnetwork/pastel-electron-wallet/pull/22/files)?

Reply:
I just moved the code from the legacy

Reply:
If this file is being copied over from legacy, it has to be refactored. Otherwise, it must stay in legacy.